docs/dictionary/command/go.lcdoc

```diff
@@ -36,6 +36,13 @@ go back 7
 Example:
 go invisible stack "Preferences"
 
+Example:
+put the filename of me into tStackFile
+set the itemDelimiter to "/"
+put item 1 to -2 of tStackFile into tStackFolder
+set the defaultFolder to tStackFolder
+go stack somethingElse.livecode
```

Comment:
It might be a good idea to add `the effective` to the first statement. That would cover cases when you are working in a substack:
    put the effective filename of me into tStackFile